Sources/SwiftParser/StringLiterals.swift

```diff
@@ -147,7 +147,7 @@ extension Parser {
     return RawTokenSyntax(
       kind: token.tokenKind,
       text: SyntaxText(rebasing: token.tokenText.dropFirst(reclassifyLeading.count).dropLast(reclassifyTrailing.count)),
-      leadingTriviaPieces: token.leadingTriviaPieces + TriviaParser.parseTrivia(reclassifyLeading, position: .trailing),
+      leadingTriviaPieces: token.leadingTriviaPieces + TriviaParser.parseTrivia(reclassifyLeading, position: .leading),
```

Comment:
@ahoppen I found this one. I tought it was a bug when reading

Reply:
Yes, seems like a bug to me. I don’t see a reason why this should be `trailing`.

Reply:
Should we add a test case for this bug?

Reply:
I you have an idea how to write one: Sure. If you don’t find a test case that exposes it 🤷🏽